client/signup/jetpack-connect/index.jsx

```diff
@@ -133,57 +162,38 @@ const JetpackConnectMain = React.createClass( {
 		} else {
 			this.checkUrl( this.state.currentUrl );
```

Comment:
@sirreal we need the sites to be loaded in order to check stuff from the site in case it's already in the sites list - see https://github.com/Automattic/wp-calypso/blob/master/client/state/jetpack-connect/actions.js#L80

This saves some requests that we perform if the site is not in a sites list - inspect the `checkUrl` action to see how it works.